apps/remix-ide-e2e/src/tests/file_explorer_multiselect.test.ts

```diff
@@ -0,0 +1,29 @@
+import { NightwatchBrowser } from 'nightwatch'
+import init from '../helpers/init'
+
+module.exports = {
+  before: function (browser: NightwatchBrowser, done: VoidFunction) {
+    init(browser, done)
+  },
+
+  'Should select multiple items in file explorer #group1': function (browser: NightwatchBrowser) {
+    const selectedElements = []
+    browser
+      .openFile('contracts')
+    browser
```

Comment:
I think you don't need to write `browser.` on each line

Reply:
You are right sort of. I still need an extra browser.